src/client.cpp

```diff
@@ -927,7 +927,7 @@ void CClient::Init()
     vecZeros.Init ( iStereoBlockSizeSam, 0 );
     vecsStereoSndCrdMuteStream.Init ( iStereoBlockSizeSam );
 
-    fMuteOutStreamGain = 1.0f;
+    // fMuteOutStreamGain = 1.0f;
```

Comment:
If this PR is merged, the line should be removed completely (not just commented). git has all the history already.
Should probably condense the two blank lines to one then.